release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -68,6 +68,22 @@ For more information, see xref:../installing/installing-troubleshooting.adoc#ins
 [id="ocp-4-12-ibm-power"]
 === IBM Power
 
+[id="ocp-4-12-images"]
+=== Images
+
+A new import value, `importMode`, has been added to the to the `importPolicy` parameter of image streams. The following fields are available for this value: 
+
+* `Legacy`: `Legacy` is the default value for `importMode`. When active,the manifest list is discarded, and a single sub-manifest is imported. 
+The platform is chosen in the following order of priority: 
++
+. Tag annotations
+. Control plane arch/OS 
+. Linux/AMD64
+. The first manifest in the list 
+
+* `PreserveOriginal`: When active, the original manifest is preserved. For manifest lists, the manifest list and all of its sub-manifests are imported. 
```

Comment:
I wasn't sure about the hyphen here. IBM SG has general guidance to avoid hyphens with prefixes or suffixes, but RH SG says to use hyphens if clarity is needed. I couldn't find any uses of the word (with or without hyphen) in the existing docs.